pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go

```diff
@@ -192,11 +194,24 @@ func (r *ReconcileAWSFederatedAccountAccess) Reconcile(request reconcile.Request
 		return reconcile.Result{}, err
 	}
 
+	configMap, err := controllerutils.GetOperatorConfigMap(r.client)
+	if err != nil {
+		log.Error(err, "failed retrieving configmap")
+	}
+	config.IsFedramp, err = controllerutils.IsFedramp(configMap)
+	if err != nil {
+		log.Error(err, "Unable to verify if cluster is fedramp")
+	}
+
 	// Get aws client
+	awsRegion := v1alpha1.AwsUSEastOneRegion
+	if config.IsFedramp {
+		awsRegion = v1alpha1.AwsUSGovEastOneRegion
+	}
```

Comment:
Can we reduce the amount of `if config.IsFedramp` change region occurances?
E.g. we could make a function `config.GetDefaultRegion()` which would return `v1alpha1.AwsUSGovEastOneRegion` if fedramp and `v1alpha1.AwsUSEastOneRegion` if not.

Reply:
done